ci/ray_ci/tester_container.py

```diff
@@ -245,6 +255,8 @@ def _run_tests_in_docker(
             test_cmd += f"--test_env {env} "
         if test_arg:
             test_cmd += f"--test_arg {test_arg} "
+        if run_flaky_tests:
+            test_cmd += f"--runs_per_test {RUN_PER_FLAKY_TEST} "
```

Comment:
this will overwrite bazel's flaky test treatment. beforehand, we are treating every test as a flaky test. meaning, when a test fails, it will auto retry, and run the test for at most 3 times, unless there is a passing run.

now this will always run all the tests for twice and only twice no matter what. it will make flaky tests occur more as failures.

are you sure this is what you want?

Reply:
i think it will run the test 6 times at most, but will report 2 accumulated test results, or maybe i misunderstand how this flag works

Reply:
let me test this locally

Reply:
i tested this locally, intentionally make the test fails

> bazel test --flaky_test_attempts=3 --runs_per_test 2 //ci/ray_ci/automation:test_determine_microcheck_tests

and it runs 6 times

> //ci/ray_ci/automation:test_determine_microcheck_tests                   FAILED in 6 out of 6 in 4.2s

Reply:
it's doing what i expect in ray environment as well: https://buildkite.com/ray-project/postmerge/builds/4380#018f5db3-0ffa-42ef-8940-324a65c6f0b0/179-4140

Reply:
hmm.. okay, I was wrong then.